src/Microsoft.Health.Fhir.Api/Features/ActionResults/JobResult.cs

```diff
@@ -0,0 +1,60 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using EnsureThat;
+using Hl7.Fhir.Model;
+using Microsoft.Health.Fhir.Core.Extensions;
+using Microsoft.Health.Fhir.Core.Models;
+
+namespace Microsoft.Health.Fhir.Api.Features.ActionResults
+{
+    public class JobResult : ResourceActionResult<Parameters>
```

Comment:
Does Parameters go back to the user? Should we consider using (Bundle.Entry.Response) from Batch-Response bundle type from https://www.hl7.org/fhir/async-bundle.html#3.2.6.2.4.0.3?

Reply:
A Batch-Response could be used to wrap the Parameters object, but the fields in Bundle.Entry.Response wouldn't allow me to send back all the relevant data.
I was trying to base this on our response for a reindex job.

Reply:
Makes sense, though reindex is the one we invented and not following spec guidelines :)